### ISSUE TITLE
Can't start OpenCTI

### ISSUE THREAD

**Issue by Rickozono** [OpenCTI-Platform/opencti#636]
After I configure production.json, with opencti and dependecies credentials, when I execute yarn serv, this is the output:
![Error](https://user-images.githubusercontent.com/51711170/79789259-494a0000-8320-11ea-8ded-38b6ff8e1e36.JPG)

**Comment by 2xyo** [comment]
Try:
```console 
$ cd /opt/opencti/src/python
$ pip3 install -r requirements.txt
$ cd /opt/opencti/ && yarn serv
```
Or:
```console 
$ cd /opt/opencti/
$ yarn install 
$ yarn build
$ yarn serv
```

**Comment by Rickozono** [comment]
I tried both ways but the output is still the same

**Comment by Rickozono** [comment]
I solved it installing Python 3.6.9 instead of 3.8.2.
Thanks for your help!
